osquery/database/plugins/rocksdb.cpp

```diff
@@ -243,6 +259,22 @@ void RocksDBDatabasePlugin::close() {
   }
 }
 
+void RocksDBDatabasePlugin::repairDB() {
+  // ROCKSDB_LITE does not have a RepairDB method. No option but to delete the
+  // corrupted DB
+  LOG(INFO) << "repairDB: Deleting corrupted db files";
+  std::vector<std::string> file_names;
+  auto s = listFilesInDirectory(path_, file_names);
+  if (s.ok()) {
+    for (auto file : file_names) {
+      fs::path file_path(file);
+      osquery::remove(file_path);
+    }
+  } else {
+    LOG(INFO) << "Unable to list " << path_ << ". Error " << s.toString();
```

Comment:
Can you combine this info message into a single phase, without punctuation? For example, "Unable to list /path/to/db: STATUS_STRING".